dev/Repeater/TestUI/Samples/FlexboxWrapPage.xaml

```diff
@@ -0,0 +1,143 @@
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+    <ScrollViewer> 
+    <StackPanel>
+        <StackPanel.Resources>
+            <Style TargetType="TextBlock">
+                <Setter Property="Foreground" Value="White"/>
+                <Setter Property="VerticalAlignment" Value="Center"/>
+                <Setter Property="HorizontalAlignment" Value="Center"/>
+                <Setter Property="FontWeight" Value="Bold"/>
+                <Setter Property="FontSize" Value="24"/>
+            </Style>
+            <Style TargetType="Grid">
+                <Setter Property="Background" Value="Orange"/>
+                <Setter Property="MinWidth" Value="120"/>
+                <Setter Property="MinHeight" Value="50"/>
+                <Setter Property="Margin" Value="6"/>
+                <Setter Property="CornerRadius" Value="6"/>
+                <Setter Property="BorderBrush" Value="Black"/>
+                <Setter Property="BorderThickness" Value="2"/>
+            </Style>
+            <Style TargetType="controls:LayoutPanel">
+                <Setter Property="Background" Value="Purple"/>
+                <Setter Property="Margin" Value="0,0,0,20"/>
+                <Setter Property="HorizontalAlignment" Value="Left"/>
+            </Style>
+            <Style x:Key="HeaderStyle" TargetType="TextBlock">
+                <Setter Property="Foreground" Value="Black"/>
+                <Setter Property="HorizontalAlignment" Value="Left"/>
+            </Style>
+                <Style  x:Key="WebviewStyle" TargetType="controls:WebView2">
+                    <Setter Property="MinHeight" Value="700"></Setter>
+                </Style>
+        </StackPanel.Resources>
+
+        <TextBlock Style="{StaticResource HeaderStyle}">NoWrap</TextBlock>
+
+        <controls:LayoutPanel MaxWidth="500">
+            <controls:LayoutPanel.Layout>
+                <controls:FlexboxLayout Direction="Row" Wrap="NoWrap"/>
+            </controls:LayoutPanel.Layout>
+            <Grid MinWidth="60">
+                <TextBlock>1</TextBlock>
+            </Grid>
+            <Grid MinWidth="60">
+                <TextBlock>2</TextBlock>
+            </Grid>
+            <Grid MinWidth="100">
+                <TextBlock>3</TextBlock>
+            </Grid>
+            <Grid MinWidth="120">
+                <TextBlock>4</TextBlock>
+            </Grid>
+            <Grid MinWidth="120">
+                <TextBlock>5</TextBlock>
+            </Grid>
+            <Grid MinWidth="60">
+                <TextBlock>6</TextBlock>
+            </Grid>
+            <Grid MinWidth="60">
+                <TextBlock>7</TextBlock>
+            </Grid>
+            <Grid MinWidth="100">
+                <TextBlock>8</TextBlock>
+            </Grid>
+        </controls:LayoutPanel>
+
+            <controls:WebView2 Source="https://codepen.io/matthewmachuca/pen/poLLeGM" Style="{StaticResource }" ></controls:WebView2>
```

Comment:
is this webview showing anything? if so, do we need the style tag?

Reply:
The webview displays a visual test comparison. it links to a codepenn of html/css example layout. it's currently my solution to having a side by side test.

the style makes the webview large enough to view